plugin/core/sessions.py

```diff
@@ -354,8 +359,12 @@ class SessionBufferProtocol(Protocol):
 
     session = None  # type: Session
     session_views = None  # type: WeakSet[SessionViewProtocol]
-    file_name = None  # type: str
-    language_id = None  # type: str
```

Comment:
I may have broken some helper package with this, need to preserve the behavior just in case.